src/Microsoft.Diagnostics.Monitoring.RestServer/Models/EventPipeConfigurationModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization;
+
+namespace Microsoft.Diagnostics.Monitoring.RestServer.Models
+{
+    [DataContract(Name = "EventPipeConfiguration")]
```

Comment:
Note that we already took a dependency on Systme.Text.Json. I think you should feel free to use those attributes (JsonElement iirc) and serializer.

Reply:
(Nvm that may not work right with the Asp.net handler)